src/lib/server/auth/email.ts

```diff
@@ -53,7 +56,15 @@ export async function deleteEmailOTP(id: string): Promise<void> {
 }
 
 export function sendOTPEmail(email: string, code: string): void {
-	console.log(`To ${email}: Your login code is ${code}`);
+	if (dev) {
```

Comment:
This is a preference question, but I would personally like to log the email exactly as we would send it (mby in the object format we give to Mailgun?). So we don't necessarily lose any information in dev, but only the transfer medium is different